blog/2015/08/join-mentoring-subcommittee.html

```diff
@@ -0,0 +1,39 @@
+---
+layout: blog
+root: ../../..
+author: Raniere Silva
+title: "Join the Mentoring Subcommittee"
+date: 2015-08-22
+time: "10:00:00"
+category: ["Community"]
+---
+<!-- start excerpt -->
+<p>
+  The Mentoring Subcommittee is seeking for new members
+  to help with the debriefing sessions and future activities.
+  If you are interested in join us, please send a email to
+  <a href="mailto:mentoring@lists.software-carpentry.org">mentoring@lists.software-carpentry.org</a>.
+</p>
+<!-- end excerpt -->
+<p>
+  Currently members of the Mentoring Subcommittee
+  spend around two hours each week with subcommittee activities
+  that includes
+</p>
+<ul>
+  <li>attend two subcommittee meetings of one hour per month,</li>
+  <li>host two debriefing sessions of one hour per month,</li>
+  <li>write blog post about the debriefing sessions for the community, and</li>
+  <li>one-to-one call with others instructors upon request.</li>
+</ul>
+<p>
+  New members of the Mentoring Subcommittee
+  will help currently members in the activities mention above
+  and engage in future activities (more on that in the upcoming months).
+</p>
+<p>
+  If you have doubts about join the Mentoring Subcommittee,
+  keep in mind that this is a great opportunity
+  to virtually meet with members of our community
+  and see it grow.
```

Comment:
I would change this paragraph to:

This is a great opportunity to meet members of our community and help it grow. We hope you'll want to take part.